spec/services/search/integration/tag_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Compare ES search to PG search for Tag", type: :feature do
```

Comment:
This is the first of a possible bunch of "integration" tests that might help us tracking differences between ES and PG results

Reply:
We should always keep in mind that we're willing to accept certain differences as long as the new results are generally sensible. If for example a few documents in the top 10 results aren't listed in the exact same order, I wouldn't necessarily consider that a problem (except for maybe the feed, where we do a lot of extra work to establish a certain order).